package.json

```diff
@@ -18,6 +18,7 @@
 	"license": "MIT",
 	"dependencies": {
 		"chalk": "^4.1.2",
+		"cosmiconfig": "^7.0.1",
```

Comment:
Lets hope the tool finds new owners and does not get abandoned.

Reply:
I agree, it's a bit of a risky move, but it's such a useful tool with lots of GitHub stars (quite popular), so IMO it has pretty good chances to survive.